charts/plane-enterprise/templates/workloads/api.deployment.yaml

```
@@ -39,6 +39,64 @@ spec:
+        - name: ca-certificates
+          emptyDir: {}
+      initContainers:
+      - name: install-ca-certs
+        image: ubuntu:22.04
+        command:
+          - /bin/bash
+          - -c
+          - |
+            set -e
+            echo "Installing custom CA certificates..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)